From 8893879e6354bee1ef7ecad3929cf7180f59a99d Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@linux.intel.com>
Date: Fri, 15 Jun 2012 13:37:57 -0700
Subject: [PATCH] Support sysconfdir in the makefile

---
 CMakeLists.txt |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 070182e..836c6a6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,6 +7,11 @@ SET(PREFIX ${CMAKE_INSTALL_PREFIX})
 SET(EXEC_PREFIX "\${prefix}")
 SET(LIBDIR "\${prefix}/lib")
 SET(INCLUDEDIR "\${prefix}/include")
+if (DEFINED SYSCONFDIR)
+	SET(SYSCONFDIR ${SYSCONFDIR})
+else()
+	SET(SYSCONFDIR ${CMAKE_INSTALL_PREFIX}/etc)
+endif()
 
 # Set required packages
 INCLUDE(FindPkgConfig)
@@ -72,7 +77,7 @@ ADD_DEPENDENCIES(${PROJECT_NAME} ps-iface-service-glue.h)
 ADD_DEPENDENCIES(${PROJECT_NAME} ps-iface-context-glue.h)
 
 # install
-INSTALL(FILES ${CMAKE_SOURCE_DIR}/src/ps-plugin.conf DESTINATION ${PREFIX}/etc/dbus-1/system.d)
+INSTALL(FILES ${CMAKE_SOURCE_DIR}/src/ps-plugin.conf DESTINATION ${SYSCONFDIR}/dbus-1/system.d)
 INSTALL(FILES ${CMAKE_SOURCE_DIR}/resources/dnet_db.sql DESTINATION /tmp)
 INSTALL(FILES ${CMAKE_SOURCE_DIR}/resources/dnet_db_data.sql DESTINATION /tmp)
 INSTALL(TARGETS ps-plugin LIBRARY DESTINATION lib/telephony/plugins)
-- 
1.7.10.3

